app/views/layouts/_survey_banner.html.haml

```diff
@@ -0,0 +1,16 @@
+.grid-row
+  .column-full
+    .survey.moj-banner
+      .message
+        <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="40" width="40">
+        <path d="M13.7,18.5h-2.4v-2.4h2.4V18.5z M12.5,13.7c-0.7,0-1.2-0.5-1.2-1.2V7.7c0-0.7,0.5-1.2,1.2-1.2s1.2,0.5,1.2,1.2v4.8
+        C13.7,13.2,13.2,13.7,12.5,13.7z M12.5,0.5c-6.6,0-12,5.4-12,12s5.4,12,12,12s12-5.4,12-12S19.1,0.5,12.5,0.5z" /></svg>
+
+        How important is People Finder for you? Please tell us your thoughts before 5 November 2019: <a href="https://www.research.net/r/D32RHGZ"> <br>
+        Take our 2 minute survey</a>
+
+:css
+  .moj-banner { border: 4px solid #1d70b8; padding: 2rem; margin-top: 2rem; margin-bottom: 2rem }
+  .moj-banner__icon { color: #1d70b8; vertical-align: middle; padding-right: 8px }
+  .message { text-indent: -52px;  margin-left: 52px; }
+  .message a { font-weight: bold }
```

Comment:
Can this go in a `survey.scss` file rather than inline for consistency?

Reply:
this stuff is being reverted in ten days' time, so the reason that the css was inline was to minimise the footprint for the change and reversion